docs/UserGuide.md

```diff
@@ -146,25 +146,68 @@ Examples:
 * `cfind John` returns `john` and `Johnathon Doe`
 * `cfind alex david` returns `Alex Yeoh` and `David Li`
 
-### Deleting a person : `delete`
+### Editing an event : `eedit`
 
-Deletes the specified person from the address book.
+Edits an existing event in the event list.
 
-Format: `delete INDEX`
+Format: `eedit INDEX [n/NAME] [at/START_TIME] [end/END_TIME] [d/DESCRIPTION] [a/ADDRESS] [z/ZOOM] [dt/TAG_DELETED]…​ [t/TAG_ADDED]…​`
 
-* Deletes the person at the specified `INDEX`.
-* The index refers to the index number shown in the displayed person list.
-* The index **must be a positive integer** 1, 2, 3, …​
+
+:information_source: **Note:** Start time should be of format “dd-MM-yyyy HH:mm” (date-month-year Hour:minutes in 24 hr format).
+
+* `elist` followed by `eedit` 2 edits the 2nd event from the event list.
+* Edits the event at the specified `INDEX`. `INDEX` refers to the index number shown in the displayed event list. `INDEX` **must be a positive integer** eg 1, 2, 3, …​
+* **At least one** of the optional fields must be provided.
+* Existing values will be updated to the input values.
+* use `t/` to add a tag.
+* use `dt/` to remove a tag.
+* use `dt/*` to remove **all** tags
+* When editing tags, the tags to be deleted will be removed first, before new tags are added
 
 Examples:
-* `list` followed by `delete 2` deletes the 2nd person in the address book.
-* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+* `elist` followed by `eedit 2 n/CS2103T Exam dt/Easy_exams t/Hard_exams` changes the name and tag of the 2nd event on the event list to `CS2103T Exam` and `Hard_exams` respectively.
+* `efind Betsy` followed by `eedit 2 n/Betsy’s Wedding` edits the name of the 2nd event from the results of the `efind` command to be `Betsy’s Wedding`
+* `eedit 3 dt/TA`  deletes the `TA` tag from the 3rd event.
+* `eedit 4 dt/*` deletes all tags from the 4th event.
+* `eedit 5 dt/Easy_exams t/Hard_exams` first deletes the `Easy_exams` tag, then adds `Hard_exams` tag to the 5th event.
```

Comment:
Don't need so many examples. line 170 and 172 can be removed and still be clear.